chat-service/index.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express'
+import cors from 'cors'
+import { createServer } from 'http'
+import { Server } from "socket.io"
+
+const app = express();
+app.use(express.urlencoded({ extended: true }))
+app.use(express.json())
+app.use(cors())
+app.options('*', cors())
+
+const server = createServer(app)
+
+const io = new Server(server, {
+    cors: {
+        origin: '*'
+    },
+})
+
+const CHAT_BOT = 'ChatBot'; // Constants and Magic Numbers will be abstracted when finalising
+io.on('connection', (socket) => {
+    console.log(`User connected ${socket.id}`)
```

Comment:
Logging: User connected to socket `${socket.id}` may be clearer instead

Reply:
Fixed